packages/synapse-react-client/src/lib/containers/access_requirement_list/AccessRequirementList.tsx

```diff
@@ -271,12 +278,17 @@ export default function AccessRequirementList({
 
   const content = (
     <>
-      <ReactBootstrap.Modal.Header closeButton={true}>
-        <ReactBootstrap.Modal.Title className="AccessRequirementList__title">
+      <DialogTitle>
```

Comment:
Replace Bootstrap 4 components with MUI so we can get rid of the `bootstrap-4-backport` class, which breaks the TextField styling